packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -112,11 +113,15 @@ export const OutboundAction = {
     type: ActionType.SortBy,
     payload: { column },
   }),
+
+  flattenRows: (): OutboundShipmentAction => ({ type: ActionType.Flatten }),
+  groupRows: (): OutboundShipmentAction => ({ type: ActionType.Group }),
 };
 
 export interface OutboundShipmentStateShape {
   draft: OutboundShipment;
   sortBy: SortBy<OutboundShipmentSummaryItem>;
+  isGrouped: boolean;
```

Comment:
I did have an action to set this explicitly, but then only used it on load, so thought it wasn't worth it 🤷